index.js

```diff
@@ -30,6 +29,11 @@ const VC = Buffer.from([0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00])
 const CRYPTO_PROVIDE = Buffer.from([0x00, 0x00, 0x01, 0x02])
 const CRYPTO_SELECT = Buffer.from([0x00, 0x00, 0x00, 0x02]) // always try to choose RC4 encryption instead of plaintext
 
+function xor (a, b) {
+  for (let i = 0, len = a.length; i < len; i++) a[i] = a[i] ^ b[i]
```

Comment:
Shouldn't we use `len = Math.max(a.length, b.length)`?

Reply:
They are always equal length? didn't see it as necessary